recipe/conda_build_config.yaml

```diff
@@ -1,11 +1,9 @@
 MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - '10.14'  # [osx and x86_64]
-  - '10.15'  # [osx and arm64]
+  - '10.15'  # [osx and x86_64]
```

Comment:
1. this is the same as in the base cbc.yaml and 2. you've taken the specification out for arm64 too, is that intentional?

Reply:
Yeah, it was intentional but I'll remove this cbc.yaml allowing the base cbc.yaml to take versions